hookbook.sh

```diff
@@ -23,23 +23,26 @@ __hookbook_shell="$(\ps -p $$ | \awk 'NR > 1 { sub(/^-/, "", $4); print $4 }')"
 __hookbook_shellname="$(basename "${__hookbook_shell}")"
 
 __hookbook_array_contains() {
-  local seeking=$1; shift
-  local in=1
-  for element; do
-    if [[ $element == $seeking ]]; then
-      in=0
-      break
-    fi
-  done
-  return $in
+  [[ $# -lt 2 ]] && \return 1
+  \local seeking=$1; \shift
+  \local check=$1; \shift
+  [[ $seeking == $check ]] && \return 0
+  __hookbook_array_contains $seeking "$@"
```

Comment:
trivial style point: write variables as `"${seeking}"`, except one-character automatic variables, which should be `"$1"`